Resources/Prototypes/Entities/Structures/Power/Generation/Singularity/singularity.yml

```diff
@@ -50,3 +53,33 @@
   - type: WarpPoint
     follow: true
     location: singularity
+  - type: Sprite
+    sprite: Structures/Power/Generation/Singularity/singularity_1.rsi
+    layers:
+    - map: [ "VisualLevel" ]
+      state: singularity_1
+  - type: GenericVisualizer
+    visuals:
+      enum.SingularityAppearanceKeys.Singularity:
+       VisualLevel:
+          1:
+            sprite: Structures/Power/Generation/Singularity/singularity_1.rsi
```

Comment:
You dont have to specify the sprite each time

Reply:
there is one sprite for each power level, how would i not change the sprite?

Reply:
yeah, in this case it does need to specify the sprite every time